src/runtime/runtime.js

```diff
@@ -537,6 +552,10 @@ export class ConfiguredRuntime {
       this.analyticsService_.logEvent(AnalyticsEvent.EVENT_PAYMENT_FAILED);
       this.jserror_.error('Redirect error', error);
     });
+
+    if (doneConstructingResolve != null) {
```

Comment:
i dont see dontConstrcutingResolve being defined. did you mean to removethis?
I think putting event manager at the top and propensity as soon as possible is good for now.

Reply:
I forgot to stage that change, thanks for noticing!